### PR TITLE
Add 'Hide empty models' filter

### PR DESCRIPTION
**Pull request by vvscode**
Improvement for https://github.com/emberjs/ember-inspector/issues/455

### CODE REVIEW

File: app/controllers/model-types.js [hunk 1]
```diff
@@ -1,9 +1,22 @@
...
+
+  filtered: filter('model', function(typeItem) {
+    let hideEmptyModels = this.get('options.hideEmptyModels');
+
+    if (hideEmptyModels) {
+      return !!Ember.get(typeItem, 'count');
+    } else {
+      return true;
+    }
+  }).property('content', 'options.hideEmptyModels')
```
Comment (teddyzeenny) [inline app/controllers/model-types.js]: s/content/property

File: app/controllers/model-types.js [hunk 2]
```diff
@@ -1,9 +1,22 @@
...
+  },
 
-  sorted: sort('model', 'sortProperties')
+  sorted: sort('filtered', 'sortProperties'),
+
+  filtered: filter('model', function(typeItem) {
+    let hideEmptyModels = this.get('options.hideEmptyModels');
+
+    if (hideEmptyModels) {
+      return !!Ember.get(typeItem, 'count');
```
Comment (teddyzeenny) [inline app/controllers/model-types.js]: Let's define `get` at the top of the file (as if we're importing it).
```js
// line 3
const get = Ember.get;
```

**Comment by teddyzeenny** [comment]
Thanks! Looks good.

Since we're calling the classes "Model Types" here can you rename `model` to `model type` where it's mentioned?

File: app/controllers/model-types.js (hunk 1 above)
Comment (vvscode) [inline app/controllers/model-types.js]: Thx for your notice

File: app/controllers/model-types.js (hunk 2 above)
Comment (vvscode) [inline app/controllers/model-types.js]: Sure. And I used it instead  this.get too

**Comment by vvscode** [comment]
@teddyzeenny  Thanks for correction

**Comment by teddyzeenny** [comment]
:+1: Can you squash the commits?

**Comment by vvscode** [comment]
@teddyzeenny  sure

**Pull request merged**